test/module/Entra/Get-EntraDirectoryRoleMember.Tests.ps1

```diff
@@ -0,0 +1,93 @@
+                    "mobilePhone"                      = "425-555-0101"
+                    "onPremisesProvisioningErrors"     = @{}
+                    "businessPhones"                   = @("425-555-0100")
+                    "Parameters"                       = $args
+                }
+            )
+        }
+    }
+    
+    Mock -CommandName Invoke-GraphRequest -MockWith $scriptblock -ModuleName Microsoft.Graph.Entra
+
+}
+
+Describe "EntraDirectoryRoleMember" {
+    Context "Test for EntraDirectoryRoleMember" {
+        It "Should return specific directory rolemember" {
+            $result = (Get-EntraDirectoryRoleMember -ObjectId "aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb") | ConvertTo-json | ConvertFrom-json
+            $result | Should -Not -BeNullOrEmpty
+            $result.ObjectId | should -Be "00aa00aa-bb11-cc22-dd33-44ee44ee44ee"
+
+            Should -Invoke -CommandName Invoke-GraphRequest -ModuleName Microsoft.Graph.Entra -Times 1
+        }
+        It "Should fail when ObjectId is empty" {
+            { Get-EntraDirectoryRoleMember -ObjectId } | Should -Throw "Missing an argument for parameter 'ObjectId'*"
+        }
+        It "Should fail when ObjectId is invalid" {
+            { Get-EntraDirectoryRoleMember -ObjectId "" } | Should -Throw "Cannot bind argument to parameter 'ObjectId' because it is an empty string.*"
+        }
+        It "Result should Contain Alias property" {
+            $result = Get-EntraDirectoryRoleMember -ObjectId "aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb"
+            $result.ObjectId | should -Be "00aa00aa-bb11-cc22-dd33-44ee44ee44ee"
+            $result.DirSyncEnabled | should -Be $null
+            $result.LastDirSyncTime | should -Be $null
+            $result.Mobile | should -Be "425-555-0101"
+            $result.ProvisioningErrors | Should -BeNullOrEmpty
+            $result.TelephoneNumber | should -Be "425-555-0100"
+        }
+        It "Should contain ObjectId in URI" {
+            $result = Get-EntraDirectoryRoleMember -ObjectId "aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb" 
+            $params = Get-Parameters -data $result.Parameters
+            $Para= $params | ConvertTo-json | ConvertFrom-Json
+            $Para.URI | Should -match "aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb"
+        }
+        It "Property parameter should work" {
+            $result = Get-EntraDirectoryRoleMember -ObjectId "aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb" -Property Id
+            $result | Should -Not -BeNullOrEmpty
+            $result.Id | Should -Be '00aa00aa-bb11-cc22-dd33-44ee44ee44ee'
+
+            Should -Invoke -CommandName Invoke-GraphRequest -ModuleName Microsoft.Graph.Entra -Times 1
+        }
+        It "Should fail when Property is empty" {
+             { Get-EntraDirectoryRoleMember -ObjectId "aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb" -Property } | Should -Throw "Missing an argument for parameter 'Property'*"
+        }
+        
+        It "Should contain 'User-Agent' header" {
```

Comment:
Update the `User-Agent` tests to look like this
https://github.com/microsoftgraph/entra-powershell/blob/86e55e47c47a0cb99444a38bf6db950084df65cd/test/module/Entra/Get-EntraUser.Tests.ps1#L72-#L84

Reply:
Changes done.